ios/RNRadar.m

```diff
@@ -31,7 +29,7 @@ - (dispatch_queue_t)methodQueue {
 }
 
 - (NSArray<NSString *> *)supportedEvents {
-    return @[@"events", @"location", @"error"];
+    return @[@"events", @"location", @"clientLocation", @"error"];
```

Comment:
what's the difference between `location` and `clientLocation` here?

Reply:
`location` is for the `didUpdateLocation` callback (synced to server), `clientLocation` is for the `didUpdateClientLocation` callback (updated on client but not necessarily synced to server)